packages/frontend-vitals-monitoring/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@govuk-one-login/frontend-vital-signs",
+  "version": "0.0.1",
+  "description": "",
+  "type": "module",
+  "esnext": "src/index.ts",
+  "main": "cjs/index.cjs",
+  "module": "esm/index.js",
+  "scripts": {
+    "build": "rollup -c",
+    "test": "jest -c"
+  },
+  "author": "Nick Heal",
```

Comment:
Author's set to `DI Frontend Capability Team` in our other repos

Reply:
Ah thank you! I'll need to test this out once merged, and I kick off the CD pipeline